modules/comment/comment.service.js

```diff
@@ -44,5 +46,49 @@ class CommentsService {
     }
     return deletedComment;
   }
+
+  async addRate(id, data) {
```

Comment:
I think we don't need to have two methods for rating, one method `rateProduct` is enough, for the client it is much easier to perform the operation via one method

Reply:
fixed